### DISCUSSION TITLE
Issue on Catchup Kodi Matrix

### DISCUSSION THREAD

**Discussion by tiritchi** [willforde/urlquick#15] [Q&A]
Hi, 

I found this topic : https://www.gitmemory.com/issue/willforde/urlquick/13/787094636
This topic explained exactly my current issue on Kodi Matrix when I try to open a 6play content. 

Do you know anything about this issue ? I'm reaaly stuck here ^^ 

thanks 

BR

**Comment by willforde** [comment]
Yeah. I have fixed that a few days ago and was waiting a few days for wwark to run tests before I submit it to kodi.
He got back to me today and all is good, so I will be pushing the fix later tonight.

**Reply by tiritchi** [reply]
Ohh wonderful. 

Nice thanks for your answer. 

Do you know approx. when Kodi will release this fix after you fix it ? 

Thks ;)
